tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,12 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
+    assert item_a in fatimah.inventory
+    assert item_b in fatimah.inventory
+    assert item_c in fatimah.inventory
```

Comment:
Is there a way to do this with a `loop` maybe? But if done with a `loop` what would be the `time complexity of this this functionality?